lib/mpl_toolkits/axisartist/angle_helper.py

```diff
@@ -250,7 +245,7 @@ def _get_number_fraction(self, factor):
                 break
 
             d = factor // threshold
-            int_log_d = int(floor(math.log10(d)))
+            int_log_d = int(math.floor(math.log10(d)))
```

Comment:
Earlier it looked like you were eliminating the use of the math module in favor of numpy functions--but not here. Why?  (The math module is faster; I assumed you were simplifying by eliminating it in contexts where the speed difference doesn't matter.)

Reply:
because the rest of the expression used math and I was too lazy to change both.  Obviously this is not on a critical path, changed.